test_scripts/API/ButtonSubscription/026_SubscribeButton_App1_App2_on_the_same_button.lua

```diff
@@ -0,0 +1,80 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0192-button_subscription_response_from_hmi.md
+-- User story: TBD
+-- Use case: TBD
+--
+-- Requirement summary: TBD
+--
+-- Description:
+-- In case:
+-- 1) Mobile app1 requests SubscribeButton(button_1)
+-- 2) Mobile app2 requests SubscribeButton(button_1)
+-- SDL does:
+-- 1) Sends Buttons.SubscribeButton(button_1, appId1) to HMI
+-- 2) Sends Buttons.SubscribeButton(button_1, appId2) to HMI
+-- 3) Process successful responses from HMI
+-- 4) Respond SubscribeButton(SUCCESS) to mobile app
+-- 5) Send OnHashChange with updated hashId to mobile app1 and app2 after adding subscription
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local common = require('test_scripts/API/ButtonSubscription/common_buttonSubscription')
+
+--[[ Test Configuration ]]
+runner.testSettings.isSelfIncluded = false
+config.application1.registerAppInterfaceParams.appHMIType = { "MEDIA" }
+config.application2.registerAppInterfaceParams.appHMIType = { "NAVIGATION" }
+
+--[[ Local Variables ]]
+local buttonName = "PRESET_0"
+
+--[[ Local functions ]]
+local function hmiLeveltoLimited(pAppId)
+  common.getHMIConnection(pAppId):SendNotification("BasicCommunication.OnAppDeactivated",
+      { appID = common.getHMIAppId(pAppId) })
+  common.getMobileSession(pAppId):ExpectNotification("OnHMIStatus",
+    { hmiLevel = "LIMITED", audioStreamingState = "AUDIBLE", systemContext = "MAIN" })
+end
+
+local function buttonPress(pButtonName)
```

Comment:
Notifications will be sent only to one app, so you can use common.buttonPress with some extension, like:
```
local function buttonPress2Apps(pAppIdToReceive, pAppId2, pButtonName)
  common.buttonPress(pAppIdToReceive, pButtonName)
  common.getHMIConnection(pAppId2):SendNotification("Buttons.OnButtonEvent")
  :Times(0)
  common.getHMIConnection(pAppId2):SendNotification("Buttons.OnButtonPress")
  :Times(0)
end
```